lib/src/main/scala/code/chymyst/jc/ReactionSite.scala

```diff
@@ -134,13 +132,13 @@ private[jc] final class ReactionSite(reactions: Seq[Reaction], reactionPool: Poo
     // Compute error messages here in case we will need them later.
     val blockingMoleculesWithNoReply = usedInputs
       .filter(_._2.reactionSentNoReply)
-      .keys.toSeq.toOptionSeq.map(_.map(_.toString).sorted.mkString(", "))
+      .map(_._1).toSeq.toOptionSeq.map(_.map(_.toString).sorted.mkString(", "))
```

Comment:
style: Most Scala developers most of the time consider avoiding using ._1 and ._2 and use intermediate case classes or pattern match to avoid them.